doc/source/dev/contributor/adding_notebooks.rst

```diff
@@ -0,0 +1,146 @@
+.. _adding-notebooks:
+
+Adding or editing tutorials as Jupyter notebooks
+------------------------------------------------
+
+Under the ``doc/source/`` folder of the SciPy tree you can find a few
+documents written in MyST markdown format. The content in these files is
```

Comment:
```suggestion
documents written in MyST Markdown format. The content in these files is
```
For consistency with how it is mentioned slightly later in the file and in other places in the docs